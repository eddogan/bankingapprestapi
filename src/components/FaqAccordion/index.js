import React, { useState, Fragment, useContext } from "react";
// Material-UI imports
import {
  Paper,
  Typography,
  Box,
  ExpansionPanel,
  ExpansionPanelDetails,
  ExpansionPanelSummary
} from "@material-ui/core";
import { makeStyles } from "@material-ui/core/styles";
// Material-UI icon imports
import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
import { BrandingContext } from "../Branding";

export default function FaqAccordion(props) {
  const brand = useContext(BrandingContext);
  const faqList = require(`./FaqLists/faq.${brand.brandingIdentifier}.json`);
  const classes = useStyles();

  const [expanded, setExpanded] = useState(false);
  const handleChange = panel => (event, isExpanded) => {
    setExpanded(isExpanded ? panel : false);
  };

  const faqs = Object.entries(faqList).map(function(faq, row) {
    const panelKey = "panel" + faq[1].id;
    const paragraphs = faq[1].descriptions.map(function(text, i) {
      return <p key={i}>{text.text}</p>;
    });
    let lists = [];
    if (faq[1].listItems) {
      lists = faq[1].listItems.map(function(text, i) {
        return <p key={i}>{text.text}</p>;
      });
    } else {
      lists = null;
    }
    return (
      <Box key={row} py={1} className="flex alignItemsCenter">
        <ExpansionPanel
          expanded={expanded === panelKey}
          onChange={handleChange(panelKey)}
        >
          <ExpansionPanelSummary
            expandIcon={<ExpandMoreIcon />}
            aria-controls="panel1bh-content"
            id="panel1bh-header"
          >
            <Typography variant="h6" component="h4">
              {faq[1].title}
            </Typography>
          </ExpansionPanelSummary>
          <ExpansionPanelDetails>
            <Typography component="div">
              {paragraphs}
              {lists}
            </Typography>
          </ExpansionPanelDetails>
        </ExpansionPanel>
      </Box>
    );
  });

  return (
    <Fragment>
      <Box my={6}>
        <Paper square={true}>
          <Typography
            className="moduleTitle"
            variant="h6"
            component="h1"
            gutterBottom
          >
            <div className={classes.moduleTitle}>
              Frequently Asked Questions
            </div>
          </Typography>
          <div className="moduleContent">{faqs}</div>
        </Paper>
      </Box>
    </Fragment>
  );
}

const useStyles = makeStyles(theme => ({
  moduleTitle: {
    backgroundColor: theme.palette.primary.main,
    color: theme.palette.primary.light
  }
}));
