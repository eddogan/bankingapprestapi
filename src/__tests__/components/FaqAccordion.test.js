import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import FaqAccordion from "../../components/FaqAccordion";

const brand = "globalclientsolutions";

it("renders correctly", () => {
  const tree = renderer.create(<FaqAccordion brand={brand} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<FaqAccordion brand={brand} />, div);
  ReactDOM.unmountComponentAtNode(div);
});
