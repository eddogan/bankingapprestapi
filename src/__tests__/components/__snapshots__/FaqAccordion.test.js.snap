// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-3"
>
  <div
    className="MuiPaper-root MuiPaper-elevation1"
  >
    <h1
      className="MuiTypography-root moduleTitle MuiTypography-h6 MuiTypography-gutterBottom"
    >
      <div
        className="makeStyles-moduleTitle-1"
      >
        Frequently Asked Questions
      </div>
    </h1>
    <div
      className="moduleContent"
    >
      <div
        className="MuiBox-root MuiBox-root-61 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                Who is Global Client Solutions?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        Global Client Solutions (GCS) is the service provider for your dedicated account. Our duties include drafting funds from your primary bank account into your dedicated account as provided for in your Application, and executing disbursement instructions in a commercially reasonable manner as soon as practical after receipt of such instructions.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-99 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                How do I contact Global Client Solutions?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        
                      </p>
                      <p>
                        GCS offices are located at 4343 S. 118th E. Ave., Suite 220 Tulsa, OK 74146.
                      </p>
                      <p>
                        Our Customer Support team can be reached by phone at
                      </p>
                      <p>
                        or by sending an email to.
                      </p>
                      <p>
                        Our offices are open Monday through Friday, from 9:00 AM to 6:00 PM Central Standard Time. Our offices are closed on all Bank Holidays.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-100 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                How do I keep track of my Account activity?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        Following your first deposit, you will receive your first monthly statement showing your Account activity and balance by mail.  All other statements will be made available online. If you desire to continue receiving a paper statement via the United States Postal Service, please contact Customer Service toll free at
                      </p>
                      <p>
                        Additionally you may access your account activity via our telephone Interactive System IVR. Telephone access requires that you use your 4-digit IVR Password to access Account information. Additionally, your IVR Password may be used for verification purposes should you need to contact a Customer Service representative. To retrieve this password, please contact Customer Service at
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-101 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                Who should I contact regarding changes to my debt program?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        You should contact your program sponsor. It is important to remember that GCS does not maintain records of your individual debts, therefore any questions regarding the negotiation of debts and the status of your debt program should be directed to your debt settlement company.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-102 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                How do I deal with creditors contacting or harassing me?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        We recommend that you contact your Debt Settlement Company with any questions that pertain to creditor calls, correspondence, and negotiations.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-103 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                How do I make a change to an automatic scheduled debit from my checking/savings account?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        If you need to make a change to an upcoming scheduled debit, contact your program sponsor who can advise you of any impact this change may have on pending settlement negotiations. Your program sponsor may provide you with additional instructions to contact our offices. You may do so by contacting our Customer Support Team at
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-104 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                Where are my funds being held on deposit?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        Your Account balance is on deposit at an FDIC-insured bank and is FDIC-insured up to $250,000 or such other limit that may be determined by the FDIC from time to time.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiBox-root MuiBox-root-105 flex alignItemsCenter"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 MuiExpansionPanel-root MuiExpansionPanel-rounded MuiPaper-rounded"
        >
          <div
            aria-controls="panel1bh-content"
            aria-disabled={false}
            aria-expanded={false}
            className="MuiButtonBase-root MuiExpansionPanelSummary-root"
            id="panel1bh-header"
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex={0}
          >
            <div
              className="MuiExpansionPanelSummary-content"
            >
              <h4
                className="MuiTypography-root MuiTypography-h6"
              >
                How do I deposit additional funds into my account?
              </h4>
            </div>
            <div
              aria-disabled={false}
              aria-hidden={true}
              className="MuiButtonBase-root MuiIconButton-root MuiExpansionPanelSummary-expandIcon MuiIconButton-edgeEnd"
              onBlur={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              role="button"
              tabIndex={-1}
            >
              <span
                className="MuiIconButton-label"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                  />
                </svg>
              </span>
            </div>
          </div>
          <div
            className="MuiCollapse-container MuiCollapse-hidden"
            style={
              Object {
                "minHeight": "0px",
              }
            }
          >
            <div
              className="MuiCollapse-wrapper"
            >
              <div
                className="MuiCollapse-wrapperInner"
              >
                <div
                  aria-labelledby="panel1bh-header"
                  id="panel1bh-content"
                  role="region"
                >
                  <div
                    className="MuiExpansionPanelDetails-root"
                  >
                    <div
                      className="MuiTypography-root MuiTypography-body1"
                    >
                      <p>
                        Global Client Solutions clients may deposit funds into their account using several methods:
                      </p>
                      <p>
                        Details for each of these deposit methods are specific to your depository bank and are included in your Welcome Letter. If you should have any questions or require additional information you may contact our Customer Support Team at
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
